objects/tickets/tagged_tickets.rb

```diff
@@ -37,31 +32,31 @@ def notify(issue, message)
   end
 
   def submit(puzzle)
-    done = @tickets.submit(puzzle)
-    issue = done[:number]
-    yaml = @sources.config
+    issue = @tickets.submit(puzzle)
+    issue_id = issue[:number]
+    yaml = @vcs.repo.config
     if yaml['tags']&.is_a?(Array)
       tags = yaml['tags'].map(&:strip).map(&:downcase)
-      labels = @github.labels(@repo)
-        .map { |json| json['name'] }
+      labels = @vcs.labels
+        .map { |json| json[:name] }
         .map(&:strip).map(&:downcase)
       needed = tags - labels
       begin
-        needed.each { |t| @github.add_label(@repo, t, 'F74219') }
-        @github.add_labels_to_an_issue(@repo, issue, tags)
-      rescue Octokit::NotFound => e
-        @github.add_comment(
-          @repo, issue,
-          "I can't create GitHub labels `#{needed.join('`, `')}`. \
-Most likely I don't have necessary permissions to `#{@repo}` repository. \
+        needed.each { |t| @vcs.add_label(t, 'F74219') }
+        @vcs.add_labels_to_an_issue(issue_id, tags)
+      rescue StandardError => e
+        @vcs.add_comment(
+          issue_id,
+          "I can't create #{@vcs.name} labels `#{needed.join('`, `')}`. \
+Most likely I don't have necessary permissions to `#{@vcs.repo.name}` repository. \
 Please, make sure @0pdd user is in the \
-[list of collaborators](https://github.com/#{@repo}/settings/collaboration):\
+[list of collaborators](#{@vcs.collaborators_link}):\
 \n\n```#{e.class.name}\n#{e.message}\n#{e.backtrace.join("\n")}\n```"
         )
-      rescue Octokit::Error => e
-        @github.add_comment(
-          @repo, issue,
-          "For some reason I wasn't able to add GitHub labels \
+      rescue StandardError => e
```

Comment:
@mbao01 it's a BAD practice. we try to catch as little as possible, not as much a possible